examples/musicPlayer/src/3_useMediaPlayer.tsx

```diff
@@ -0,0 +1,108 @@
+import { usePlayMedia } from "@/lib/audio/usePlayMedia";
+import { usePlayState } from "@/lib/audio/usePlayState";
+import { useAccount, useCoState } from "@/lib/jazz";
+import { MusicTrack, Playlist } from "@/1_schema";
+import { BinaryCoStream } from "jazz-tools";
+import { useEffect } from "react";
+
+export function useMediaPlayer() {
+    const { me } = useAccount();
+
+    const activeTrack = useCoState(MusicTrack, me?.root?.activeTrack?.id, {
+        file: [],
```

Comment:
This depth definition was required to have a non-null value on the `file` attribute.
Also passing an array here doesn't look right, but TS forced me into it.

Is this intentional?